OneCommunityGlobal/HGNRest
# Iven - Allow accounts with permission make suggest on tasks and see the notification after approval

## Related PRS (if any):
To test this backend PR you need to checkout the [#2200](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2200) frontend PR.

## Main changes explained:
- Update `taskHelper.js`, fix the logic in building taskNotifications list in each user's data, also fix a potential NaN error in timeEntry to pass the lint


## How to test:
1. check into current branch
2. do `npm install` 
3. do `npm run build`
4. do `npm start`
5. following the instruction in frontend PR
